code/game/gamemodes/gameticker.dm

```diff
@@ -151,6 +175,17 @@ var/global/datum/controller/gameticker/ticker
 
 	callHook("roundstart")
 
+	//Here we will trigger the auto-observe and auto bst debug things
```

Comment:
This seems hacky. What is this "auto observe" and "auto BST" stuff?

Reply:
Debug settings, mentioned in the description